codes/data/multi_country/get_adm_info.py

```diff
@@ -408,23 +411,26 @@ def main():
     us_pops["adm0_name"] = "USA"
     us_pops = us_pops.set_index(["adm0_name", "adm1_name", "adm2_name"])
 
-
     # ##### Merge back into global adm datasets
 
     adm2_gdf = adm2_gdf.fillna(us_pops)
     st_pops = (
-        adm2_gdf.loc[:, "population"].groupby(["adm0_name", "adm1_name"]).sum(min_count=1)
+        adm2_gdf.loc[:, "population"]
+        .groupby(["adm0_name", "adm1_name"])
+        .sum(min_count=1)
```

Comment:
does this work?

Reply:
With no evidence nor knowledge my heart feels like the dots should go the line above

Reply:
it works! FWIW this was using `black` an autoformatter. I've never seen it actually break things b/c it's pretty conservative but it makes your code look nice and pretty. You just call `black path_to_file`